packages/react/src/exchanges/hooks/types/useExchange.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  Config,
+  ExchangeBookRequest,
+  PostExchangeData,
+  PostExchangeFilterData,
+} from '@farfetch/blackout-client';
+
+export type UseExchangeOptions = {
+  enableAutoFetch?: boolean;
+  fetchConfig?: Config;
+  exchangeData?: PostExchangeData;
```

Comment:
Both `exchangeData` and `exchangeFilterData` should not be passed as options as they are only used in the `create` actions but the user can pass it directly. It would be useful to add those parameters if the hook would do something else with them.

Reply:
Fixed!